queries/dune_v2/period_slippage.sql

```diff
@@ -231,12 +231,13 @@ potential_buffer_trades as (
              token,
              block_time
              -- exclude 0 to prevent zero division, and exclude very small values for performance
-    having abs(sum(amount)) > 0.0001
-),
-valued_potential_buffered_trades as (
+    having sum(amount) != cast(0 as int256)
```

Comment:
We're not excluding small values for performance here anymore. If this is intentional, we should adjust the comment above.

Reply:
AND this is probably the reason we were seeing the row counts disagreeing...

Reply:
The thing here is that these amounts are now integers so 0.00001 doesn't mean anything anymore.